practices/graded-browser-support.md

```diff
@@ -83,6 +83,16 @@ Unfortunately, both IE10 and IE11 support the same CSS media queries, which mean
 
 This means that even if we consider IE10 a Grade-C browser, we currently serve IE10 users the full experience.
 
+#### Internet Explorer and CSS Grid
+
+[CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) has been availalable in all evergreen browsers [for the last year](https://caniuse.com/#feat=css-grid). It offers a much-simplified way of working with layout and offers new design opportunities.
+
+However, CSS Grid is not available in the full standardised form in Internet Explorer 10 or 11, both of which are currently *de facto* supported by us (see above).
```

Comment:
I'm not sure about the wording of this sentence, specifically:

> "both of which are currently [...] supported by us"

Quoting @dotcode (from memory): "bear in mind we support all browsers, if not at Grade-A, then at Grade-X or Grade-C". I think we have been more or less consistent in following this use of the word "support" so far in the playbook (with the caveat of effectively not supporting browsers without TLS 1.2+ out of necessity).

That said, I'm don't have a better alternative to suggest at the moment :(